examples/event_handler_rest/src/accessing_request_details.py

```diff
@@ -16,12 +16,12 @@
 @app.get("/todos")
 @tracer.capture_method
 def get_todos():
-    todo_id: str = app.current_event.get_query_string_value(name="id", default_value="")
+    todo_id: str = app.current_event.query_string_parameters["id"]
```

Comment:
How about using `.get()`? So we prevent the customer from having problems with `KeyError`.